cterasdk/edge/shares.py

```diff
@@ -49,7 +49,7 @@ def add(self,
         param.acl = []
         for entry in acl:
```

Comment:
Since the default value for acl is None, this method can crash here. Since this method does not do anything if acl is empty, you can just check it and return in the beginning. Another option is to add: "acl = acl or []". This will ensure that acl is a list (if it was None). Please note that you must not set acl to an empty list in the default parameters since setting a mutable value as a default will create issues later